README.md

```diff
@@ -13,3 +13,146 @@
   complexity suggests a different time.
 
 Add your answers to this markdown file.
+
+
+//
+
+
+Name: Kane Kriz
+
+Start Date: 11 Feb 2025
+
+Feedback Request 1: 12 Feb 2025
+
+
+//
+
+
+Feedback Request 2: 11 April 2025
+
+
+//
+
+(I added the bolded titles in to better separate the reasoning of the issues from one another as in the original feedback request they were kind of bleeding into one another and making it hard to follow my reasoning.
+Also to make it more evident that reason 2 is not the same as reason 1, which was one of the outlined issues from the edits requested.
+Thank you.)
+
+
+Response 1: 
+
+
+**Disregard for lower terms**
+
+The first issue with asymptotic analysis being potentially misleading is due to a fundamental part of asymptotic analysis as a whole.
+Asymptotic analysis focuses upon the highest order and most important term in regards to performance.
+
+However, smaller order terms still make an impact in realistic scenarios without notably large (asymptotic) input sizes.
+In varying situations, these smaller order terms still can make a difference to the overall performance and runtime, though with asymptotic analysis, they are intentionally ignored.
+
+
+
+
+
+
+**Oversimplified Algorithm groupings by highest order terms**
+
+Another potential issue with asymptotic analysis is that it groups algorithms only by their highest order terms.
+This varies from the first provided reason as this considers the grouping by highest order terms as not harmful through the ignorance of lower order values, but due to oversimplification of grouping algorithms by these highest order terms.
+
+This could give the impression that two algorithms that have the same asymptotic complexity in common will perform similarly.
+In actuality, real world performance has the potential to vary dramatically due to more nuanced factors that are disregarded when simply considering asymptotics and shared highest order.
+
+The primary issue is that asymptotic analysis describes behavior as n grows extremely large, but practical inputs exist at limited scales, such as the mentioned 1,000 or 10,000 elements within the exercise/problem description.
+At these sizes, lower order terms (which are ignored at large in asymptotic analysis) can significantly impact an algorithm's performance.
+
+
+
+
+
+**Defined Case Types being Restrictive**
+
+A third misleading issue about asymptotic analysis is that one of three scenarios are always considered: worst case, average case, or best case scenario.
+Real world inputs often fall outside these best, middle, and worst case categories.
```

Comment:
Not sure what you mean here -- it can't be worse than the worst case or better than the best case.